apps/assisted-disconnected-ui/src/components/Wizard.tsx

```diff
@@ -67,6 +67,7 @@ const EditCluster = ({ clusterId }: { clusterId: string }) => {
                     cluster={cluster}
                     infraEnv={infraEnv}
                     updateInfraEnv={updateInfraEnv}
+                    isDissconnectedMode={true}
                   />
                 </ClusterWizardContextProvider>
```

Comment:
you should change the `ClusterWizardContextProvider` to accept `isDissconnectedMode`.